napari/layers/labels/_tests/test_labels.py

```diff
@@ -251,6 +251,22 @@ def test_colormap():
     assert type(layer.colormap[1]) == Colormap
 
 
+def test_custom_color_dict():
+    """Test custom color dict."""
+    np.random.seed(0)
+    data = np.random.randint(20, size=(10, 15))
+    layer = Labels(data, color_dict={1: 'white'})
+
+    # test with custom color dict
+    assert type(layer.get_color(2)) == np.ndarray
+    assert type(layer.get_color(1)) == np.ndarray
+    assert (layer.get_color(1) == np.array([1.0, 1.0, 1.0, 1.0])).all()
+
+    # test disable custom color dict
+    layer.color_mode = 'direct'
+    assert not (layer.get_color(1) == np.array([1.0, 1.0, 1.0, 1.0])).all()
```

Comment:
given that we have an `np.random.seed` call in this function we know that it won't happen for this example, so I think it is fine to leave as is, but you could @ziyangczi add a comment about this in case one day someone else looks at this and wonders the same thing